docker/postgres-techempower-fortunesbench/run.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+if (( $# != 1 ))
+then
+    echo "Usage: run.sh server-name-or-ip"
+    exit 1
+fi
+
+host=$1
+threads=$(nproc --all)
+connections=$(($threads * 8))
+
+#echo on
+set -x
+
+docker run \
+    -it \
```

Comment:
Though the people looking at this most of the time will probably know it already ;)